www/locations/nash/index.html

```diff
@@ -6,9 +6,9 @@ <h1>PyLadies Nashville</h1>
             <article>
                 <ul class="social">
                     <aside>
-                        <h4><a href="{{ get_url("locations/nash") }}">Nashville, TN</a> | <a class="social icon vcard" data-icon="&#59170;" href="mailto:info@pyladies.com" title"Contact"></a><a class="social icon twitter" data-icon="&#62217;" title="Twitter" href="https://twitter.com/PyNashLadies"></a><a class="social icon location" data-icon="&#59172;" title="Meetup Link" href="http://www.meetup.com/Nashville-PyLadies"></a></h4>
+                        <h4><a href="{{ get_url("locations/nash") }}">Nashville, TN</a> | <a class="social icon vcard" data-icon="&#59170;" href="https://github.com/pyladies/info/issues/new" title"Contact"></a><a class="social icon twitter" data-icon="&#62217;" title="Twitter" href="https://twitter.com/PyNashLadies"></a><a class="social icon location" data-icon="&#59172;" title="Meetup Link" href="http://www.meetup.com/Nashville-PyLadies"></a></h4>
```

Comment:
```suggestion
                        <h4><a href="{{ get_url("locations/nash") }}">Nashville, TN</a> | <a class="social icon twitter" data-icon="&#62217;" title="Twitter" href="https://twitter.com/PyNashLadies"></a><a class="social icon location" data-icon="&#59172;" title="Meetup Link" href="http://www.meetup.com/Nashville-PyLadies"></a></h4>
```

Reply:
I don't think this chapter exists anymore. Anyway, we can keep the listing, but I rather we don't get asked about chapter that doesn't exist! So I suggest to remove just the "contact" link, and keep the rest.

Reply:
@Mariatta this is probably better done as a second PR -- remove all the inactive chapters from pyladies website ... I've been stating we'd be doing it since Nov 2019 ...

Reply:
I agree on removing the inactive chapters on a later PR.